searx/settings.yml

```diff
@@ -1407,6 +1407,25 @@ engines:
   #      WHERE title LIKE :wildcard OR description LIKE :wildcard
   #      ORDER BY duration DESC
 
+  - name: tagesschau
+    engine: json_engine
+    paging: true
+    search_url: https://www.tagesschau.de/api2u/search?searchText={query}&resultPage={pageno}
```

Comment:
Seems the API description at https://github.com/AndreasFischer1985/tagesschau-api/blob/69362cd840085ff02fa03ef4db899b304a783145/openapi.yaml#L107-L143 .. is a bit misleading: "description: Suchergebnisse pro Seite (1-30)" .. the `resultPage` starts with `0` while SearXNG's `pageno` starts with `1` / compare:

- https://www.tagesschau.de/api2u/search?searchText={query}&pageSize=10&resultPage=0

The API is very powerful, we can use much more data from it if we implement a Python module for it.

Can I persuade you to implement a Python module for this? ... it would be enough if you make a basic implementation ... the details I can implement then.
- https://www.tagesschau.de/api2u/search?searchText={query}&pageSize=10&resultPage=1